types/mercadopago-sdk-js/core.d.ts

```diff
@@ -0,0 +1,161 @@
+  status: string;
+};
+
+type Result = {
+  financial_institutions: unknown[];
+  secure_thumbnail: string;
+  payer_costs: PayerCost[];
+  issuer: Issuer;
+  total_financial_cost?: unknown;
+  min_accreditation_days: number;
+  max_accreditation_days: number;
+  merchant_account_id: string;
+  id: string;
+  payment_type_id: string;
+  accreditation_time: number;
+  thumbnail: string;
+  bins: unknown[];
+  marketplace: string;
+  deferred_capture: string;
+  agreements: unknown[];
+  labels: string[];
+  financing_deals: FinancingDeals;
+  name: string;
+  site_id: string;
+  processing_mode: string;
+  additional_info_needed: string[];
+  status: string;
+  settings: Setting[];
+};
+
+type Paging = {
+  total: number;
+  limit: number;
+  offset: number;
+};
+
+type PaymentMethodsResponse = {
+  paging: Paging;
+  results: Result[];
+};
+
+type IssuersParams = {
+  payment_method_id?: string;
+  paymentMethodId?: string;
+  bin: string;
+};
+
+type IssuersResponse = {
+  id: string;
+  name: string;
+  secure_thumbnail: string;
+  thumbnail: string;
+  processing_mode: string;
+  merchant_account_id: string;
+};
+
+type InstallmentsParams = {
+  locale?: string;
+  amount: string;
+  bin: string;
+  paymentMethodId?: string;
+  payment_method_id?: string;
+  processingMode?: ProcessingMode;
+  processing_mode?: ProcessingMode;
+  paymentTypeId?: string;
+  payment_type_id?: string;
+};
+
+type InstallmentsResponse = {
+  payment_method_id: string;
+  payment_type_id: string;
+  issuer: Issuer;
+  processing_mode: string;
+  merchant_account_id?: string;
+  payer_costs: PayerCost[];
+  agreements?: unknown;
+};
+
+type CardTokenParams = {
+  cardNumber?: string;
+  cardholderName?: string;
+  identificationType?: string;
+  identificationNumber?: string;
+  securityCode?: string;
+  cardExpirationMonth?: string;
+  cardExpirationYear?: string;
+  cardId?: string;
+};
+
+type CardTokenUpdateParams = {
+  securityCode?: string;
+  cardExpirationMonth?: string;
+  cardExpirationYear?: string;
+  token?: string;
+};
+
+type ValidateFieldsParams = {
+  cardNumber: boolean;
+  cardExpirationMonth: boolean;
+  cardExpirationYear: boolean;
+  securityCode?: boolean;
+};
+
+type MercadoPagoCore = {
+  getIdentificationTypes(): Promise<IdentificationTypesResponse[]>;
+  getPaymentMethods(
+    paymentMethodsParams: PaymentMethodsParams
+  ): Promise<PaymentMethodsResponse>;
+  getIssuers(issuersParams: IssuersParams): Promise<IssuersResponse[]>;
+  getInstallments(
+    installmentsParams: InstallmentsParams
+  ): Promise<InstallmentsResponse[]>;
+  createCardToken(
+    cardTokenParams: CardTokenParams,
+    validateFieldsParams: ValidateFieldsParams
+  ): Promise<CardTokenResponse>;
+  updateCardToken(
+    CardTokenUpdateParams: CardTokenUpdateParams,
+    validateFieldsParams: ValidateFieldsParams
+  ): Promise<CardTokenResponse>;
+
+  fields: fields;
+
+  bricks(style: BricksStyle): Bricks
+
+  cardForm(options: CardFormOptions): CardForm;
```

Comment:
Apenas fields começa com letra minúscula
Poderia começar com maiúscula para padronizar?
```suggestion
  fields: Fields;

  bricks(style: BricksStyle): Bricks

  cardForm(options: CardFormOptions): CardForm;
```

Reply:
claro! obrigado!